### ISSUE TITLE
TextInput render prop don't show full label

### ISSUE THREAD

**Issue by mMarcos208** [callstack/react-native-paper#4251]
### Current behaviour
When using TextInput with render props, label don't show well.

### Expected behaviour
Show full label

### How to reproduce?

```
<TextInput
      error={!!error}
      focusable
      label="Quantidade"
      showSoftInputOnFocus
      returnKeyType="done"
      mode="outlined"
      render={({value, onChangeText, onFocus, onBlur, style}) => (
        <TextInputMask
          maxLength={inputProps.maxLength}
          returnKeyType="done"
          editable={inputProps.editable}
          value={value}
          style={style}
          onChangeText={(text) => {
            onChangeText?.(text)
            onChange?.(text)
          }}
          onFocus={onFocus}
          onBlur={(event) => {
            inputProps.onBlur?.(event)
            onBlur?.(event)
          }}
          {...maskProps}
        />
      )}
    />
```
    
### Preview

![image](https://github.com/callstack/react-native-paper/assets/14351491/8d1bdcbb-b57a-43aa-b9d7-d14b444b05d8)

### Your Environment

| software                      | version
| --------------------- | -------
| ios                                | 17.2
| react-native                | 0.72.7
| react-native-paper    | 5.11.5
| node                            | v20.10.0
| yarn                             | 1.22.10

**Comment by mMarcos208** [comment]
Only this code.

```
<TextInput
      error={!!error}
      focusable
      label="Quantidade"
      showSoftInputOnFocus
      returnKeyType="done"
      mode="outlined"
      render={({value, onChangeText, onFocus, onBlur, style}) => (
        <TextInputMask
          maxLength={inputProps.maxLength}
          returnKeyType="done"
          editable={inputProps.editable}
          value={value}
          style={style}
          onChangeText={(text) => {
            onChangeText?.(text)
            onChange?.(text)
          }}
          onFocus={onFocus}
          onBlur={(event) => {
            inputProps.onBlur?.(event)
            onBlur?.(event)
          }}
          {...maskProps}
        />
      )}
    />
```

**Comment by lukewalczak** [comment]
Hey @mMarcos208 could you please create an expo snack?

**Comment by mMarcos208** [comment]
@lukewalczak expo example https://snack.expo.dev/@mbarroso208/curious-blue-churros

**Comment by ad-walker** [comment]
Ran into the same issue with our own outlined TextInput components when upgrading to [v5.11.0](https://github.com/callstack/react-native-paper/compare/v5.10.6...v5.11.0).

**Comment by lukewalczak** [comment]
Hey @mMarcos208, please also pass `onLayout` from render props to the `TextInputMask`, it should solve your problem:

```
      render={({value, onFocus, style, onLayout}) => (
        <TextInputMask
          returnKeyType="done"
          value={value}
          style={style}
          onFocus={onFocus}
          onLayout={onLayout}
          type="custom"
        />
      )}
 ```

**Comment by lukewalczak** [comment]
@ad-walker could you please share your issue details?

**Comment by mMarcos208** [comment]
@lukewalczak could you add this prop?

**Comment by lukewalczak** [comment]
> @lukewalczak could you add this prop?

What do you mean?

**Comment by mMarcos208** [comment]
> > @lukewalczak could you add this prop?
> 
> What do you mean?

typescript prop. I need to add ts-ignore in my file.

**Comment by lukewalczak** [comment]
I see, it should be released in `5.11.7`.
